source/_integrations/inkbird.markdown

```diff
@@ -0,0 +1,29 @@
+---
+title: Inkbird
+description: Instructions on how to integrate INKBIRD devices into Home Assistant.
+ha_category:
+  - Sensor
+ha_zeroconf: true
```

Comment:
I don't think this is correct, this is not zeroconf?

Reply:
Nope definitely not :)

That should have been ha_bluetooth!

Reply:
I'll fix it when I get back home.

Reply:
Hold on to that, I think I can solve it generically (needs other adjustments as well).

Reply:
Sorry I missed this.

Reply:
Don't worry about that, we all miss a thing or two in reviews. Stuff happens 👍

Reply:
Thankfully, mistakes can be corrected, and https://github.com/home-assistant/home-assistant.io/pull/23498